### PR TITLE
Changed displayed name for API owner in API Store to business owner w…

### PR DESCRIPTION
**Pull request by thilinicooray**
…hen business owner is available

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java [hunk 1]
```diff
@@ -1313,6 +1313,7 @@ public static NativeArray jsFunction_getRecentlyAddedAPIs(Context cx,
                 if(api.isAdvertiseOnly()){
                     currentApi.put("owner",currentApi,APIUtil.replaceEmailDomainBack(api.getApiOwner()));
                 }
+                currentApi.put("businessOwner", currentApi, APIUtil.replaceEmailDomainBack(api.getBusinessOwner()));
```
Comment (jranabahu) [inline components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java]: Please check formatting

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/responsive/templates/subscription/subscription-list-element/template.jag [hunk 2]
```diff
@@ -533,8 +533,8 @@
             			<tr>
 						 <th>API</th>
 				         <th>Owner</th>
-				         <th>Sunscription Status</th>				         
-				         <th>Sunscription</th>
+				         <th>Subscription Status</th>
```
Comment (madusankapremaratne) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/responsive/templates/subscription/subscription-list-element/template.jag]: Please use i18n

Comment (thilinicooray) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/responsive/templates/subscription/subscription-list-element/template.jag]: Fixed in PR #1033 commit e724ec8

File: components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java (hunk 1 above)
Comment (lafernando) [inline components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java]: Use constants for all string literals.

Comment (thilinicooray) [inline components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java]: Fixed in PR #1317

**Pull request merged**